ddtrace/_trace/processor/trace.py

```diff
@@ -70,6 +69,72 @@ def process_trace(self, trace):
         pass
 
 
+@attr.s
```

Comment:
why using attr dependency as we don't need to be python 2 compatible?

Reply:
I want to minimize code changes for this move.